### PR TITLE
3x tweaks

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java [hunk 1]
```diff
@@ -178,10 +178,8 @@ public GroupByNumericArrayIterator(
       this.statsCollector = tsdb.getStatsCollector();
 
       if (this.result.isSourceProcessInParallel()) {
-        logger.debug("Accumulate in parallel, source size {}", sources.size());
```
Comment (gskchaitanya) [inline core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java]: Can we keep this debug log, please? Helps during debugging

File: core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java [hunk 2]
```diff
@@ -178,10 +178,8 @@ public GroupByNumericArrayIterator(
...
         accumulateInParallel(sources, valuesCombiner);
       } else {
-        logger.debug("Accumulate in sequence, source size {}", sources.size());
```
Comment (gskchaitanya) [inline core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java]: Also this one ^

File: core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java (hunk 1 above)
Comment (manolama) [reply core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java]: How bout trace?

Comment (gskchaitanya) [reply core/src/main/java/net/opentsdb/query/processor/groupby/GroupByNumericArrayIterator.java]: We have a deal!

**Pull request merged**